modules/ROOT/pages/rtc/rtc-troubleshooting.adoc

```diff
@@ -34,13 +34,13 @@ Care must be taken to avoid losing encryption keys to this race condition.
 
 == Common JWT error messages
 
-If the below error descriptions do not solve the problem, try pasting the token into the https://jwt.io/#debugger-io[\https://jwt.io/ Debugger] for detailed token-format debugging.
+If the below error descriptions do not solve the problem, try pasting the token into the https://jwt.io/#debugger-io[https://jwt.io/ Debugger] for detailed token-format debugging.
 
 === Unsupported JWT signature algorithm
 
 RTC requires an _asymmetric signing algorithm_ for JWT identity tokens. Asymmetric signatures use separate private and public keys; the public key stored on the {cloudname} server can only be used to verify signatures, not create new ones. The private key, used to create new signatures, remains private.
 
-For a list of supported algorithms, see: link:{baseurl}/rtc/jwt-authentication/#supportedalgorithms[RTC JWT Authentication - Supported algorithms].
+For a list of supported algorithms, see: xref:rtc/jwt-authentication.adoc#supportedalgorithms[RTC JWT Authentication - Supported algorithms].
```

Comment:
This xref doesn't resolve as the `supportedalgorithms` id doesn't exist. I'm not exactly sure where this is meant to link to though and is broken in production as well... this will probably be a good one to note down to fix later.

Reply:
comment added